membership_variable_period/views/product_template_views.xml

```diff
@@ -1,74 +1,45 @@
-            <field name="model">product.template</field>
-            <field name="inherit_id" ref="membership.membership_products_form" />
-            <field name="arch" type="xml">
-                <label for="membership_date_from" position="before">
-                    <field name="membership_type"/>
-                    <label for="membership_interval_qty"
-                           string="Interval"
-                           attrs="{'invisible': [('membership_type', '!=', 'variable')]}"
-                           />
-                    <div attrs="{'invisible': [('membership_type', '!=', 'variable')]}">
-                        <field name="membership_interval_qty"
-                               attrs="{'required': [('membership_type', '=', 'variable'), ('membership', '=', True)]}"
-                               class="oe_inline"
-                               nolabel="1"
-                                />
-                        <field name="membership_interval_unit"
-                               attrs="{'required': [('membership_type', '=', 'variable'), ('membership', '=', True)]}"
-                               class="oe_inline"
-                               nolabel="1"
-                                />
-                    </div>
-                </label>
-                <label for="membership_date_from" position="attributes">
-                    <attribute name="attrs">{'invisible': [('membership_type', '!=', 'fixed')]}</attribute>
-                </label>
-                <xpath expr="//field[@name='membership_date_from']/.." position="attributes">
-                    <attribute name="attrs">{'invisible': [('membership_type', '!=', 'fixed')]}</attribute>
-                </xpath>
-                <field name="membership_date_from" position="attributes">
-                    <attribute name="attrs">{'required': [('membership_type', '=', 'fixed'), ('membership', '=', True)]}</attribute>
-                    <attribute name="required">0</attribute>
-                </field>
-                <field name="membership_date_to" position="attributes">
-                    <attribute name="attrs">{'required': [('membership_type', '=', 'fixed'), ('membership', '=', True)]}</attribute>
-                    <attribute name="required">0</attribute>
-                </field>
-            </field>
-        </record>
+<record model="ir.ui.view" id="membership_products_form_period">
+    <field name="name">Membership Products (variable period)</field>
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="membership.membership_products_form" />
+    <field name="arch" type="xml">
+        <label for="membership_date_from" position="before">
+            <field name="membership_type"/>
+            <label for="membership_interval_qty"
+                   string="Interval"
+                   attrs="{'invisible': [('membership_type', '!=', 'variable')]}"/>
```

Comment:
It's possible @chienandalu . I have not tested it.

But anyway I recreated the translations files for that module and these strings: 
```
msgid "{'invisible': [('membership_type', '!=', 'fixed')]}"
```
and these
```
msgid ""
"{'required': [('membership_type', '=', 'fixed'), ('membership', '=', True)],"
" 'invisible': [('membership_type', '!=', 'fixed')]}"
msgid "{'invisible': [('membership_type', '!=', 'fixed')]}"
```
and these:
```
msgid ""
+"{'required': [('membership_type', '=', 'fixed'), ('membership', '=', True)],"
+" 'invisible': [('membership_type', '!=', 'fixed')]}"
```
are not created.
So you have to avoid to be translated or you have and incorrect po files.